src/main/kotlin/org/janelia/saalfeldlab/labels/blocks/n5/LabelBlockLookupFromN5.kt

```diff
@@ -29,9 +29,9 @@ class LabelBlockLookupFromN5(
 
 		private const val SINGLE_ENTRY_BYTE_SIZE = 3 * 2 * java.lang.Long.BYTES
 
-		private fun fromBytes(array: ByteArray): MutableMap<Long, Array<Interval>> {
+		private fun fromBytes(array: ByteArray): TLongObjectHashMap<Array<Interval>> {
```

Comment:
I haven't done any benchmarks but thought that this wouldn't do any harm. Do you prefer `Map<Long, ..>` here because this way the key could be easily made a generic parameter?

Reply:
I generally do not like the interface of the trove collections very much and use them only when a standard collection would inhibit performance. Generally, I think it is a good strategy to use standard collections as much as possible because most code will expect standard collections.

Reply:
Switched back to standard `MutableMap` / `Map` collections